pkg/swiveltesting/exit_zctoken_filling_zctoken_initiate_test.go

```diff
@@ -166,17 +166,16 @@ func (s *EZFZISuite) TestEZFZI() {
 	assert.Equal(amt, amount)
 
 	// first call to utoken transferfrom 'from' should be maker here...
```

Comment:
this block had been commented out because of how the mocks will overwrite successive calls to the same method if the calling address is the same, with the fix in EZFZI (the msg.sender change) it allows us to check both of these sequentially now as the `transferFrom` is called with .Maker, then msg.sender (Owner.Opts.From in the test)

that is why this is uncommented now -- it occurs to me that this was actually catching the bug and i didn't realize it.